AFQ/api.py

```diff
@@ -271,9 +271,11 @@ def __init__(self,
             [VIZ] Whether to use a virtual fram buffer. This is neccessary if
             generating GIFs in a headless environment. Default: False
         viz_backend : str, optional
-            [VIZ] Which visualization backend to us.
-            One of {"fury", "plotly"}.
-            Default: "fury"
+            [VIZ] Which visualization backend to use.
+            See Visualization Backends page in documentation
```

Comment:
I believe that you can link directly to the docs from here using rst link syntax. If the link doesn't exist yet, it's pretty straightforward to infer what it will be once this is released.